{"results":[{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"LessThan\n","instructorExeTime":23,"studentStdOut":"LessThan\n","studentExeTime":3,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":139,"studentErrOut":"Seg fault","instructorStdOut":"Inbetween boundaries\n","instructorExeTime":3,"studentStdOut":"","studentExeTime":2,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":139,"studentErrOut":"Seg fault","instructorStdOut":"Inbetween boundaries\n","instructorExeTime":2,"studentStdOut":"","studentExeTime":2,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"6\n","instructorExeTime":70,"studentStdOut":"6\n","studentExeTime":72,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":1,"studentErrOut":"Exception in thread \"main\" java.lang.NumberFormatException: For input string: \"4\" under radix 2\n\tat java.base\/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)\n\tat java.base\/java.lang.Long.parseLong(Long.java:711)\n\tat Main.main(Main.java:4)\n","instructorStdOut":"Invalid value\n","instructorExeTime":40,"studentStdOut":"","studentExeTime":37,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"Stream closed","instructorStdOut":"Finished Loop\n","instructorExeTime":72,"studentStdOut":"","studentExeTime":5007,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"","instructorExeTime":73,"studentStdOut":"","studentExeTime":81,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"truefalsetruefalsetruefalsetruefalsetruefalsefalsefalsetruefalsetruefalsetruefalsetruefalse\n","instructorExeTime":76,"studentStdOut":"truefalsetruefalsetruefalsetruefalsetruefalsefalsefalsetruefalsetruefalsetruefalsetruefalse\n","studentExeTime":79,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"Exception in thread \"Thread-0\" Exception in thread \"Thread-1\" java.lang.AssertionError: RaceCondition occurred\n\tat Number.run(Module9.java:36)\njava.lang.AssertionError: RaceCondition occurred\n\tat Number.run(Module9.java:36)\n","instructorStdOut":"","instructorExeTime":39,"studentStdOut":"","studentExeTime":42,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"0\n1\nThread with Priority 0 executed\n2\n3\nThread with Priority 1 executed\nThread with Priority 2 executed\n4\nThread with Priority 3 executed\n5\nThread with Priority 4 executed\n6\nThread with Priority 5 executed\n7\n8\nThread with Priority 6 executed\nThread with Priority 7 executed\n9\nThread with Priority 8 executed\n10\nThread with Priority 9 executed\n0\n1\n2\n3\n","instructorExeTime":74,"studentStdOut":"0\n0\n1\nThread with Priority 0 executed\n0\n1\n2\nThread with Priority 1 executed\n0\n1\n2\n3\nThread with Priority 2 executed\n0\n1\n2\nThread with Priority 3 executed\n3\n","studentExeTime":84,"instructorErrOut":""}]}